tests/run_tests.sh

```diff
@@ -72,12 +76,12 @@ setup() {
 
 	set +e
 	sudo pkill --signal SIGKILL -P $(ps faux | grep 'daily' | awk '{print $2}')
-	set -e
 
 	# Remove docker daemon and containerd.
 	sudo systemctl stop docker
 	sudo systemctl stop containerd
 	sudo apt-get purge -y docker-ce docker-ce-cli containerd.io
+	set -e
```

Comment:
Why was `set -e` moved down?

Reply:
How do you running tests locally? 

My steps is as following:

1. startup vagrant
2. run `./tests/run_tests.sh` to initial the test environment
3. modify and run test case until test pass

I need to change run_tests.sh so i can initial my test environment. 

Is there a better test step there?

Reply:
aah I see your concern here 🙂
Initially, when I designed the tests, I wrote them to run only as part of a CI/CD system like `circleci`

See https://github.com/Roblox/nomad-driver-containerd/blob/master/tests/run_tests.sh#L17-L19
and 
https://github.com/Roblox/nomad-driver-containerd/blob/master/tests/run_tests.sh#L56

I will open a separate PR, so these tests can be run locally as well, inside the vagrant VM.

Reply:
We can put `set -e` back to its original place for now.

Reply:
done